lib/config/config-file.js

```diff
@@ -210,6 +244,8 @@ function loadConfigFile(filePath) {
             default:
                 config = loadLegacyConfigFile(filePath);
         }
+    } else if (filePath.indexOf("eslint-plugin") >= 0) {
```

Comment:
By the time it gets here, it already normalized the name and stripped out `plugin:` so that's why I'm looking for `eslint-plugin`